src/studio/src/designer/backend/Views/Home/Index.cshtml

```diff
@@ -1,15 +1,15 @@
 @{
   ViewBag.Title = "Altinn Studio";
 }
-<html lang="no">
+<html lang="no" style="font-size: 100%;">
```

Comment:
Wouldnt it be better to add these styles to the dashboard css file instead, to keep all the style related code together with the app code? 

https://github.com/Altinn/altinn-studio/blob/master/src/studio/src/designer/frontend/dashboard/app/App.css

Reply:
Whoops, didn't see that css file. Adding the changes there instead.